book/src/todo_2.md

```diff
@@ -196,6 +196,11 @@ Combined with [`serde_json`](https://lib.rs/crates/serde_json) we can save our t
 
 First, we extend our `Cargo.toml` with the `serde` and `serde_json` crate.
 
+```
+cargo add serde --derive
```

Comment:
That doesn't look right.

Shouldn't that be something like `--feature=derive`?

Reply:
You're right. Fixed.